python/tempo/tsdf.py

```diff
@@ -429,23 +429,23 @@ def withRangeStats(self, type='range', colsToSummarize=[], rangeBackWindowSecs=1
   def write(self, spark, tabName, optimizationCols = None):
     tio.write(self, spark, tabName, optimizationCols)
 
-  def resample(self, freq, func=None, metricCols = None, prefix=None):
+  def resample(self, freq, func=None, metricCols = None, prefix=None, fill = None):
     """
     function to upsample based on frequency and aggregate function similar to pandas
     :param freq: frequency for upsample - valid inputs are "hr", "min", "sec" corresponding to hour, minute, or second
     :param func: function used to aggregate input
```

Comment:
Can we update the documentation on the additional function params not included here:
- metricCols
- prefix
- fill

Reply:
Thanks. Done.